_includes/nav-bar.html

```diff
@@ -1,7 +1,9 @@
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Viewport to enable viewport-based unit usage. If this website is to be a long term project, you should probably move most things to use viewport units.

Reply:
viewport units are difficult to compose, i disagree

Reply:
also, this declaration doesn’t enable viewport units - those just work. what it does is make the page naturally sized to mobile screens instead of using a pseudo-desktop viewport intended to fix issues with mobile-unaware websites

Reply:
also this isn’t in head, I think, so it will do nothing.